app/controllers/diaper_drives_controller.rb

```diff
@@ -4,7 +4,11 @@ class DiaperDrivesController < ApplicationController
   # GET /diaper_drives
   # GET /diaper_drives.json
   def index
-    @diaper_drives = DiaperDrive.all
+    setup_date_range_picker
+    @diaper_drives = DiaperDrive.class_filter(filter_params)
+    @diaper_drives = @diaper_drives.within_date_rage(@selected_date_range)
+    @diaper_drives = @diaper_drives.order(created_at: :desc)
```

Comment:
Can we make this a single assignment instead of a longer chain?

Reply:
@mdworken This should be fixed now.